tests/test_integration.py

```diff
@@ -378,7 +378,7 @@ def test_rebuild(self):
 class CommandTest(IGVMTest):
     def setUp(self):
         super(CommandTest, self).setUp()
-        vm_build(VM_HOSTNAME)
+        vm_build(VM_HOSTNAME, rebuild=True)
```

Comment:
I'm surprised this was not necessary on BuildTest too. Did you get lucky just because of order of tests?

Reply:
If so, we got lucky every single time. Also some of them have rebuild set to true.

Reply:
It failed again on a random test. The only one with `rebuid=True` is `test_rebuild()`. We should not write code depending on particular order of tests and as far as I understand the 1st test can't be run as rebuild because it will not be able to undefine the VM.

Reply:
Is this still needed since we clean cert before each test?